### PR TITLE
Colors fixed, button text in single mode fixed

### PR DESCRIPTION
**Pull request by lukaszwo**

### CODE REVIEW

**Comment by MasterAccountIFM** [comment]
--- JIRA LINKS ---

File: filepicker/src/main/java/com/github/angads25/filepicker/view/FilePickerDialog.java [hunk 1]
```diff
@@ -151,34 +153,38 @@ public void notifyCheckBoxIsClicked() {
                  *  selected.
                  */
                 positiveBtnNameStr = positiveBtnNameStr == null ?
-                        context.getResources().getString(R.string.choose_button_label) : positiveBtnNameStr;
+                                             context.getResources().getString(R.string.choose_button_label)
```
Comment (rainqer) [inline filepicker/src/main/java/com/github/angads25/filepicker/view/FilePickerDialog.java]: very strange formatting

File: filepicker/src/main/java/com/github/angads25/filepicker/view/FilePickerDialog.java [hunk 2]
```diff
@@ -431,11 +427,22 @@ public void show() {
         } else {
             super.show();
             positiveBtnNameStr = positiveBtnNameStr == null ?
-                    context.getResources().getString(R.string.choose_button_label) : positiveBtnNameStr;
+                    getString(R.string.choose_button_label) : positiveBtnNameStr;
```
Comment (rainqer) [inline filepicker/src/main/java/com/github/angads25/filepicker/view/FilePickerDialog.java:430]: could one line but .... not a blocker

File: filepicker/src/main/java/com/github/angads25/filepicker/view/FilePickerDialog.java [hunk 3]
```diff
@@ -431,11 +427,22 @@ public void show() {
...
             select.setText(positiveBtnNameStr);
+            int size = MarkedItemList.getFileCount();
+            if (size == 0) {
+                select.setText(positiveBtnNameStr);
```
Comment (rainqer) [inline filepicker/src/main/java/com/github/angads25/filepicker/view/FilePickerDialog.java]: zero is a positive number ?

**Review by rainqer — CHANGES_REQUESTED** [review]

File: filepicker/src/main/java/com/github/angads25/filepicker/view/FilePickerDialog.java (hunk 2 above)
Comment (domhys) [reply filepicker/src/main/java/com/github/angads25/filepicker/view/FilePickerDialog.java:430]: it won't fit in one line :/

**Review by rainqer — APPROVED** [review]

**Pull request merged**